.github/workflows/doxygen.yml

```diff
@@ -0,0 +1,30 @@
+name: Doxygen
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  Build and Publish Documentation:
+    name: Build and Publish Documentation
+    runs-on: ubuntu-latest
+    container:
+      image: novelrt/novelrt-build:latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Generate Documentation
+        run: scripts/cibuild.sh --documentation -DDOXYGEN_OUTPUT_DIR=$GITHUB_WORKSPACE/novelrt/docs
+
```

Comment:
I added a flag to only generate documentation... this should work with and without `cibuild.sh` as the change is rooted in `build.sh`.

Should we add this for the Windows variant as well?

Reply:
I think yes. Ideally we can do any build step on either system.